src/PostgREST/Main.hs

```diff
@@ -17,12 +18,18 @@ import Network.Wai.Middleware.Static (staticPolicy, only)
 import Network.Wai.Middleware.RequestLogger (logStdout)
 import Data.List (intercalate)
 import Data.Version (versionBranch)
+import Data.Functor.Identity
+import Data.Text(unpack)
 import qualified Hasql as H
 import qualified Hasql.Postgres as P
 import Options.Applicative hiding (columns)
 
 import PostgREST.Config (AppConfig(..), argParser, corsPolicy)
 
+isServerVersionSupported = do
+  Identity row <- H.tx Nothing $ H.singleEx $ [H.stmt|SHOW server_version_num|]
+  return $ read (unpack row) > 90200
```

Comment:
Rather than using particular string conversion functions such as `unpack` I like to use the [cs](https://hackage.haskell.org/package/string-conversions-0.4/docs/Data-String-Conversions.html#v:cs) function throughout the codebase and let its return type polymorphism determine the conversion.